arm/Microsoft.Synapse/workspaces/.bicep/nested_cmk.bicep

```
@@ -0,0 +1,37 @@
+param encryptionKeyVaultName string
+param encryptionKeyVaultResourceGroupName string
+param encryptionKeyName string
+param workspaceName string
+param workspacePrincipalId string
+
+var keyVaultUrl = 'https://${encryptionKeyVaultName}${environment().suffixes.keyvaultDns}/keys/${encryptionKeyName}'
+
+resource keyVault 'Microsoft.KeyVault/vaults@2019-09-01' existing = {
+  name: encryptionKeyVaultName
+  scope: az.resourceGroup(encryptionKeyVaultResourceGroupName)
+}
+
+// Workspace encryption - Assign Synapse Workspace MSI access to encryption key
+module workspace_cmk_rbac 'nested_cmkRbac.bicep' = {
+  name: '${workspaceName}-cmk-rbac'
+  params: {
+    workspaceIdentity: workspacePrincipalId
+    keyvaultName: encryptionKeyVaultName
+    usesRbacAuthorization: keyVault.properties.enableRbacAuthorization
+  }
+  scope: az.resourceGroup(encryptionKeyVaultResourceGroupName)
```

`az` namespace is not strictly needed here as far as I can see.

```suggestion
  scope: resourceGroup(encryptionKeyVaultResourceGroupName)
```